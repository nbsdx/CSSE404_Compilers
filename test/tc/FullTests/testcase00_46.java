class foo {
	public static void main(String[] args) {
		System.out.println(bar.fib(4));
	}
}

class bar {
	public int fib(int a) {
		int result = 0;
		if (a <= 2) {
			result = 1;
		} else {
			result = bar.fib(a-1) + bar.fib(a-2);
		}
		return result;
	}
}