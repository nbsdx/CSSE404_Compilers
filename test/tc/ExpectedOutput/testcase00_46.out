Variable bar is not declared.
No method named fib found for class null
In MiniJava, System.out.println only takes an int.  The expression has type null
Variable bar is not declared.
No method named fib found for class null
Variable bar is not declared.
No method named fib found for class null
Left argument of type null does not match expected type int for operator PLUS
Right argument of type null does not match expected type int for operator PLUS
